back/spec/acceptance/omniauth_callback_spec.rb

```diff
@@ -128,4 +175,81 @@ def mock_omniauth_callback_controller(auth_service_mock)
       end
     end
   end
+
```

Comment:
Expanded test coverage for this very important controller

Reply:
nice